frontend/src/components/common/form/FastCurrencyInput.tsx

```diff
@@ -81,7 +84,7 @@ const CurrencyInput = ({
   const isInvalid = error && touch ? 'is-invalid ' : '';
   const isValid = !error && touch && value && !disabled ? 'is-valid ' : '';
   return (
-    <Form.Group className={classNames(outerClassName, 'fast-currency-input')}>
+    <Form.Group className={classNames(!!required ? 'required' : '', outerClassName)}>
```

Comment:
last 'fast-currency-input' class with this change, not sure if that is used anywhere. Please confirm.

Reply:
I'm not sure what you mean

Reply:
@devinleighsmith